manifests/package/debian.pp

```diff
@@ -14,12 +14,12 @@
 #
 # This class file is not called directly
 class nginx::package::debian (
-    $manage_repo              = true,
-    $package_name             = 'nginx',
-    $package_source           = 'nginx',
-    $package_ensure           = 'present',
-    $passenger_package_ensure = 'present'
-  ) {
+  $manage_repo              = true,
+  $package_name             = 'nginx',
+  $package_source           = 'nginx',
+  $package_ensure           = 'present',
+  $passenger_package_ensure = 'present'
+) {
```

Comment:
I don't think this is right -- it's fine to switch things to the new convention in how we're calling classes, but if I'm not mistaken, I think this is just hard-coding everything. So I think it either needs to be passed in from https://github.com/voxpupuli/puppet-nginx/blob/master/manifests/package.pp or else using `nginx::foo` directly.  https://github.com/voxpupuli/puppet-nginx/blob/master/manifests/config.pp I think kind of shows how I've been doing that in newer stuff.